web/src/app/pages/create-survey/task-details/task-input/option-editor/_option-editor.component-theme.scss

```diff
@@ -0,0 +1,22 @@
+@use 'sass:map';
```

Comment:
Please add copyright header with "The Ground Authors." as copyright owner.

Reply:
all the other _*.component-theme files haven't the copyright. can we do it in a separate pr?